smart-contracts/assembly/contracts/registry.ts

```diff
@@ -60,10 +59,10 @@ export function constructor(binaryArgs: StaticArray<u8>): void {
     .nextString()
     .expect('WmasTokenAddress is missing or invalid');
 
-  // ensure that the fee share protocol is between 0 and 1
+  // ensure that the fee share protocol is greater than 0
```

Comment:
And less than 1000 (10%). For security reasons better to put 10%, no reason more.